ansible/roles/owncloud/tasks/setup_owncloud.yml

```diff
@@ -89,8 +89,8 @@
   ansible.builtin.set_fact:
     owncloud__do_autosetup: '{{ (owncloud__autosetup and
                                  owncloud__admin_username|d() and
-                                 (owncloud__register_occ_check is not skipped) and
```

Comment:
Why remove `owncloud__register_occ_check`? I don‘t know exactly why (presumably) I added that condition but there was a reason, potentially check mode.